distribution/lib/Standard/Base/0.0.0-dev/src/System/Environment.enso

```diff
@@ -18,4 +18,36 @@ polyglot java import java.lang.System
 
          example_get = Environment.get "PATH"
 get : Text -> Text | Nothing
-get key = System.getenv key
+get key = Environment_Utils.get_environment_variable key
+
+## UNSTABLE
+
+   Returns a value of a specified environment variable or the provided default
+   value if such variable is not defined.
+
+   Arguments:
+   - key: The name of the environment variable to look up.
+   - default: The default fallback value.
+
+   > Example
+     Look up the value of the `FOOBAR` environment variable.
+
+         import Standard.Base.System.Environment
+
+         example_get_or_else = Environment.get_or_else "FOOBAR" "my default"
+get_or_else : Text -> Text -> Text
+get_or_else key ~default = case get key of
+    Nothing -> default
+    value -> value
+
+## PRIVATE
+   ADVANCED
+
+   Runs a given action with an environment variable modified to a given value.
+   The environment variable is restored to its original value after the action.
+   The environment variable override is only visible to the Enso
+   `Environment.get` method, the environment as seen from a direct
+   `System.getenv` Java call remains unchanged.
+unsafe_with_environment_override : Text -> Text -> Any -> Any
```

Comment:
Maybe this should be an extension method in Test?

Reply:
Definitely! That sounds like a much better place for it, as I was quite hesitant putting it in `Base`.

Reply:
I moved it, does it look good now?